recipes/python-pdal/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "python-pdal" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "60d41da79441d0aa9c44ec6ebb7497ffc0090d761259d42d4eeb05c7146bb432" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/PDAL/python/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py!=36]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - cython
+    - packaging
+    - pdal 1.7.*
+    - pip
+  run:
```

Comment:
[Looks like](https://github.com/PDAL/python/blob/2.0.0/setup.py#L202) you should have `numpy` and `packaging` as `run` deps too.